addons/account/models/account_journal_dashboard.py

```diff
@@ -647,6 +641,13 @@ def _get_late_payment_query(self):
             ('journal_id.type', '=', 'purchase'),
         ])
 
+    def _get_to_check_payment_query(self):
```

Comment:
I know it's a fix, minimal diff, blablabla but could be cool to use this hook function in `_fill_general_dashboard_data` as it's the same domain (if you do, probably in master)